src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IIsPatternExpression.cs

```diff
@@ -1455,6 +1457,12 @@ class D
 ";
 
             VerifyOperationTreeForTest<IsPatternExpressionSyntax>(compilation, expectedOperationTree);
+
+            var tree = compilation.SyntaxTrees[0];
+            var isPattern = tree.GetRoot().DescendantNodes().OfType<IsPatternExpressionSyntax>().Single();
+            var model = compilation.GetSemanticModel(tree);
+            var instanceReceiver = model.GetOperation(isPattern).Descendants().OfType<IInstanceReferenceOperation>().Single();
+            TestOperationVisitor.Singleton.Visit(instanceReceiver);
```

Comment:
>TestOperationVisitor.Singleton.Visit(instanceReceiver); [](start = 12, length = 55)

It is not clear to me what is this doing which VerifyOperationTreeForTest wouldn't already do. Could you please elaborate?

Reply:
`VerifyOperationTreeForTest` does not call the TestOperationVisitor on operation nodes.

---
In reply to: [528951526](https://github.com/dotnet/roslyn/pull/49534#discussion_r528951526) [](ancestors = 528951526)